apps/custom-ui-floating-window-demo/pages/api/addContact.js

```diff
@@ -0,0 +1,63 @@
+import { getCookie } from "cookies-next";
+import {
+  PersonsApi,
+  NewPerson,
+  BasePersonItemPhone,
+  NotesApi,
+  AddNoteRequest,
+} from "pipedrive";
+import logger from "../../shared/logger";
+import { initAPIClient } from "../../shared/oauth";
+const log = logger("Add Contact API ➕");
+
+export default async function handler(req, res) {
+  try {
+    log.info("Getting session details");
+    const session = getCookie("session", { req, res });
+    const d = JSON.parse(req.body);
+
+    const client = initAPIClient({
+      accessToken: JSON.parse(session).token,
+    });
```

Comment:
I see this part of code being used in multiple places, can you move it to single function, something like `getApiClient(req, res);`